Classes/common/HTTP/CDTURLSessionTask.h

```diff
@@ -0,0 +1,24 @@
+//
+//  CDTURLSessionTask.h
+//  
+//
+//  Created by Rhys Short on 20/08/2015.
+//
+//
+#import <Foundation/Foundation.h>
+#import "CDTMacros.h"
+
+@interface CDTURLSessionTask : NSObject
+
+@property (readonly) NSURLSessionTaskState state;
+@property (nonnull,readonly) NSURLSessionTask *task;
+
+- (nullable instancetype) init UNAVAILABLE_ATTRIBUTE;
+- (nullable instancetype) initWithTask:(nonnull NSURLSessionTask *) task NS_DESIGNATED_INITIALIZER;
+
+
+- (void) resume;
+- (NSURLSessionTaskState) state;
+- (void) cancel;
```

Comment:
Why not just doc that they are pass-throughs? Then I wouldn't have needed to ask the question.